Added full command tab completion and fixed bugs

itsTyrion opened the pull request:
Added command tab completion 
Fixed bugs with turkish system locale
Chopped some very long lines down
Slight clean ups

itsTyrion committed:
Added full command tab completion
Fixed bugs with turkish system locale
Chopped some very long lines down
Slight clean ups

itsTyrion committed:
Added full command tab completion
Fixed bugs with turkish system locale
Chopped some very long lines down
Slight clean ups

itsTyrion committed:
Merge remote-tracking branch 'origin/master'

Shevchik commented:
I do not accept style changes. Remove them and leave only fixes + features. Also the command tab completion and locale bug fix should be separate commits.

itsTyrion commented:
> I do not accept style changes. Remove them and leave only fixes + features. Also the command tab completion and locale bug fix should be separate commits.

why should toLowerCase() -> toLowerCase(Locale.ENGLISH) be in a different commit than tab completion? It's in the same class. But ok..

Also what do you mean by "style change"? How is "I chopped down a line that is literally so long it doesn't fit onto a 16:10 screen at font size 13" a super illegal "style" change?

itsTyrion committed:
Added full command tab completion
Fixed bugs with turkish system locale
Chopped some very long lines down
Slight clean ups

itsTyrion committed:
Synchronize directly on uncompleted events field

itsTyrion commented:
I fucked up the git history. I'll redo it [here](https://github.com/Shevchik/BungeePluginManager/pull/7)
